src/api/UserApi.tsx

```diff
@@ -54,6 +54,13 @@ const getActiveUserRoles = () => {
     return roles;
 };
 
+const getActiveUserId = () => {
+    const token = AuthStore.getToken() as string;
+    const user: any = jwt_decode(token);
+
+    return user.nameid as string;;
```

Comment:
looks like "user" variable is redundant

Reply:
I will not be able to access the variable "nameid" without it